.changeset/friendly-pears-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+"@alleyinteractive/block-editor-tools": patch
```

Comment:
imo this should be a minor version, not a patch, since we're introducing new functionality